### PR TITLE
Please add the German language

### PR DESCRIPTION
**Pull request by siggi1984**

### CODE REVIEW

**Comment by Alliba** [comment]
What did I do wrong?
What do I have to correct so that it runs?

File: app/src/main/res/values-DE/strings.xml [hunk 1]
```diff
@@ -0,0 +1,306 @@
...
+    <string name="sort_by_song_count">Anzahl der Lieder</string>
+    <string name="sort_by_length">Länge</string>
+    <string name="sort_by_play_time">Spielzeit</string>
+
+    <!-- Snackbar -->
+    <plurals name="snackbar_delete_song">
+        <item quantity="one">%d Song wurde gelöscht.</item>
+        <item quantity="other">%d Lieder wurden gelöscht.</item>
+    </plurals>
+    <plurals name="n_selected">
```
Comment (z-huang) [inline app/src/main/res/values-DE/strings.xml:221]: Need one line below:
`<item quantity="one">{string when n is 1}</item>`

**Review by z-huang — CHANGES_REQUESTED** [review]

File: app/src/main/res/values-DE/strings.xml (hunk 1 above)
Comment (Alliba) [reply app/src/main/res/values-DE/strings.xml:221]: > 

Sorry if I ask again stupid but which line exactly and what should be in the line.

Comment (Alliba) [reply app/src/main/res/values-DE/strings.xml:221]: > Need one line below: `<item quantity="one">{string when n is 1}</item>`

I have inserted it, it should be correct now.
Thank you.

**Review by z-huang — APPROVED** [review]

**Comment by z-huang** [comment]
Thanks!

**Pull request merged**